--- /home/krusty/rpmbuild/BUILD/packaging-test-v0.2/greet.sh	2020-03-30 12:48:37.000000000 +0200
+++ greet.sh	2020-07-07 20:57:52.954637972 +0200
@@ -1,5 +1,18 @@
-#!/bin/sh
+#!/bin/bash
 
-GREETING="Hello!"
+# greet.sh
+# Shell scripting test script
+# by Krystof Sara / 26. 6. 2020
+
+PRINT_N_TIMES=1
+[[ $# -gt 0 && $1 -gt 0 ]] && PRINT_N_TIMES=$1 
+
+WEEKDAYS=("Monday" "Tuesday" "Wednesday" "Thursday" "Friday" "Saturday" "Sunday")
+
+GREETING="Hello $USER! It is ${WEEKDAYS[$(date +%u)-1]} today."
+
+# Iterate over the $1 optional
+for i in $(seq 1 $PRINT_N_TIMES); do
+	echo ${GREETING};
+done
 
-echo ${GREETING}
